blockchain/storage/leveldb/leveldb.go

```diff
@@ -4,10 +4,51 @@ import (
 	"fmt"
 
 	"github.com/dogechain-lab/dogechain/blockchain/storage"
+	"github.com/dogechain-lab/dogechain/helper/common"
 	"github.com/hashicorp/go-hclog"
 	"github.com/syndtr/goleveldb/leveldb"
+	"github.com/syndtr/goleveldb/leveldb/filter"
+	leveldbopt "github.com/syndtr/goleveldb/leveldb/opt"
 )
 
+const (
+	// minCache is the minimum memory allocate to leveldb
+	// half write, half read
+	minCache = 16
```

Comment:
uint in `MiB`?

Why should we set minimum to this, not some `32` or `8`?

Reply:
Got it. Since  default `gasLimit` of `Dogechain` block is close to `Ethereum`, `16 MiB` would be a better choice.

Could you provide the unit in its comment?  It would save all contributors' time.